docs/src/main/sphinx/release/release-476.md

```diff
@@ -0,0 +1,171 @@
+
+## Druid connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## DuckDB connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## Elasticsearch connector
+
+## Exasol connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## Faker connector
+
+## Google Sheets connector
+
+* Add support for authentication using delegated user credentials using the `gsheets.delegated-user-email` config property. ({issue}`25746`)
+
+## Hive connector
+
+* Add support for excluding certain tables from the directory listing cache, controlled by a new configuration option `hive.file-status-cache.excluded-tables`.  ({issue}`25715`)
+* Add signer type to native S3 filesystem. ({issue}`25820`)
+* Improve compatibility with S3-compliant storage systems ({issue}`25791`)
+* Allow configuring User-ManagedIdentity in AzureFS. ({issue}`23447`)
+* Improve the directory listing cache invalidation behavior to avoid invalidating the entire cache when dropping a partitioned table. ({issue}`25749`)
+* Fix query failure when reading ORC files with a large row count. ({issue}`25634`)
+* Reduce S3 throttling failures by increasing streaming upload part size and reducing number of requests. ({issue}`25781`)
+* Fix query failures when attempting to read parquet data with integer logical annotation as a `date` column. ({issue}`25667`)
+
+## Hudi connector
+
+* Allow configuring User-ManagedIdentity in AzureFS. ({issue}`23447`)
+* Fix query failures when attempting to read parquet data with integer logical annotation as a `date` column. ({issue}`25667`)
+
+## Iceberg connector
+
+* Allow configuring User-ManagedIdentity in AzureFS. ({issue}`23447`)
+* Add signer type to native S3 filesystem. ({issue}`25820`)
+* Improve compatibility with S3-compliant storage systems ({issue}`25791`)
+* Show detailed metrics from splits generation in output of EXPLAIN ANALYZE VERBOSE. ({issue}`25770`)
+* Add `max_partitions_per_writer` session property, which corresponds to the `iceberg.max-partitions-per-writer` configuration property. ({issue}`25662`)
+* Improve query planning performance when reading from materialized views. ({issue}`24734`)
+* Fix rare failure when `iceberg.bucket-execution` is enabled. ({issue}`25125`)
+* Fix query failure when reading ORC files with a large row count. ({issue}`25634`)
+* Reduce S3 throttling failures by increasing streaming upload part size and reducing number of requests. ({issue}`25781`)
+* Fix query timeout errors due to concurrent writes on tables with large number of manifest files. ({issue}`24751`)
+* Fix query failures when attempting to read parquet data with integer logical annotation as a `date` column. ({issue}`25667`)
+
+## Ignite connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## JMX connector
+
+## Kafka connector
+
+## Kafka Event Listener
```

Comment:
I dont think we had that section in the past for any event listeners.. maybe check with old release notes and stay consistent

Reply:
Fair .. we should add it and other event listerns to the release notes template then.. so we know next time.